docs/kickstart-docs.rst

```diff
@@ -130,6 +130,13 @@ You could use an entry similar to the following:
 
 ``ignoredisk --drives=/dev/disk/by-path/pci-0000:00:05.0-scsi-*``
 
+However, the ``*`` pattern will fail the installation in case no device could
```

Comment:
Slight rewording:
However, the installation will fail if the ``*`` pattern does not match a device.